app/routes.py

```diff
@@ -0,0 +1,91 @@
+import json
+
+from flask import render_template
+from os import path
+
+from app import app
+from app.data import USER_TO_ADD, TASK_TO_ADD
+
+data_dir = path.join(path.dirname(__file__), 'data')
+users_data = path.join(data_dir, 'users.json')
+tasks_data = path.join(data_dir, 'tasks.json')
+
+users = json.loads(open(users_data).read())
+tasks = json.loads(open(tasks_data).read())
+
+
+@app.route('/')
+@app.route('/index')
+def index():
+    return render_template('index.html', users=users, tasks=tasks)
+
+
+@app.route('/users', methods=['GET'])
+def get_users():
+    return json.dumps(users)
+
+
+@app.route('/users', methods=['POST'])
+def add_user():
+    users.append(USER_TO_ADD)
+    return json.dumps(users)
+
+
+@app.route('/users/<user_id>', methods=['DELETE'])
+def delete_user(user_id):
+    for user in users:
+        if user['id'] == int(user_id):
+            users.remove(user)
+        else:
+            print ('My friends are gonna be there too')

```

Comment:
Strange...
You have a lot of print with 'my friends...'

Reply:
Better say "you'll", cause I won't have it, until code is runned

Reply:
:P